examples/src/main/scala/org/apache/spark/examples/streaming/ActorWordCount.scala

```diff
@@ -63,11 +63,11 @@ class FeederActor extends Actor {
   def receive: Receive = {
     case SubscribeReceiver(receiverActor: ActorRef) =>
       println("received subscribe from %s".format(receiverActor.toString))
-      receivers = LinkedList(receiverActor) ++ receivers
+      receiverActor +=: receivers
 
     case UnsubscribeReceiver(receiverActor: ActorRef) =>
       println("received unsubscribe from %s".format(receiverActor.toString))
-      receivers = receivers.dropWhile(x => x eq receiverActor)
+      receivers.dropWhile(x => x eq receiverActor)
```

Comment:
> Is there any mutation method that does this?

There are -= and --= which take an element, respectively an seq of elements to remove. Alternatively there is remove(n: Int) which takes an index to remove.

Considering the options, I would highly recommend going with a Set